.pre-commit-config.yaml

```diff
@@ -7,11 +7,11 @@ repos:
       entry: make
       args: ["tidy"]
       pass_filenames: false
-    - id: make-manifests
-      name: make-manifests
+    - id: make-bundle
```

Comment:
It makes sense as long as `manifests` is called by `bundle`.
Is this a pattern we're going to spread across the board?

Reply:
yes that would be the plan if this works to spread this same change to all repos for the default.